### DISCUSSION TITLE
Doris 2.0.0_alpha-be-x86_64 exports to S3(minio) failed with all BEs crashing

### DISCUSSION THREAD

**Discussion by dpsisodia** [apache/doris#22527] [A - General / Q&A]
Hi,
env: docker, apache/doris:2.0.0_alpha-be-x86_64, 3FE and 3BE. 40 core cpu, 200GB ram, single server node. 
When attempting to export formatted data to S3(minio) and 3 BE crashed. Below is the error seen in terminal. attempted 2 times and both time same thing happened BEs crashed. 
Query used:

set enable_parallel_outfile = true;
select * from wpg_test where CLOSED=1 and FROM_UNIXTIME (DATE_IN/1000, '%Y-%m-%d')='2023-08-02' 
into outfile "s3://wpg-test/2023-08-02-parquet"
format as parquet
properties
(
     "s3.endpoint" = "http://<validIP>:9000",
     "s3.access_key" = "aaa",
     "s3.secret_key" = "xxx",
     "s3.region" = "pun-on-prem"
)
Results in below error
ERROR 1105 (HY000): RpcException, msg: java.util.concurrent.TimeoutException: Waited 5000 milliseconds (plus 79967 nanoseconds delay) for io.grpc.stub.ClientCalls$GrpcFuture@1cb55bd[status=PENDING, info=[GrpcFuture{clientCall={delegate=ClientCallImpl{method=MethodDescriptor{fullMethodName=doris.PBackendService/exec_plan_fragment_prepare, type=UNARY, idempotent=false, safe=false, sampledToLocalTracing=true, requestMarshaller=io.grpc.protobuf.lite.ProtoLiteUtils$MessageMarshaller@39aea7b1, responseMarshaller=io.grpc.protobuf.li

**Comment by LemonLiTree** [comment]
is be down？Look at the log of be.out
